backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/nudger/BpkNudger.kt

```diff
@@ -104,6 +109,69 @@ fun BpkNudger(
     }
 }
 
+@Composable
+fun BpkNudger(
+    value: Int,
+    onValueChange: (Int) -> Unit,
+    min: Int,
+    max: Int,
+    title: String,
+    modifier: Modifier = Modifier,
+    subtitle: String? = null,
+    icon: BpkIcon? = null,
+    enabled: Boolean = LocalFieldStatus.current != BpkFieldStatus.Disabled,
+) {
+    Row(
+        modifier = modifier
+            .semantics {
```

Comment:
we should avoid combining strings like this as it's error prone. we should be able to just rely on `mergeDescendants` here